howso/utilities/testing.py

```diff
@@ -66,9 +66,17 @@ def get_configurationless_test_client(
             k: v for k, v in os.environ.items()
             if k not in names_to_remove
         }
+
+        if kwargs.get("trace"):
+            amlg_options = kwargs.get("amalgam", {})
+            amlg_options.setdefault("execution_trace_dir", "./traces")
+            kwargs["amalgam"] = amlg_options
+
         # Once we stop supporting 3.9 and lower, we can parenthesize these.
         # See: https://docs.python.org/3.10/whatsnew/3.10.html#parenthesized-context-managers
-        with patch('howso.client.client.get_configuration_path') as mocked_fn, \
-             patch.dict('os.environ', modified_environ, clear=True):
+        with (
```

Comment:
I think the immediately preceding comment about parenthesized context managers still applies?  We still target Python 3.9 and this syntax isn't necessarily legal?